lib/forms.js

```diff
@@ -11,55 +11,136 @@ var deleteAppReferences = require('./impl/deleteAppRefrences.js');
 var dataSourceImpl = require('./impl/dataSources');
 var dataTargetImpl = require('./impl/dataTargets');
 var config = require('./config');
+var mongoUriParser = require('mongodb-uri');
 var logger = logfns.getLogger();
+var async = require('async');
+
+//Single mongoose connection for forms.
+var mongooseConnection;
+var dbConnection;
+
+//TODO: This should be global.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1135)